### PR TITLE
fix issue 53

### PR DESCRIPTION
**Pull request by londumas**
https://github.com/igmhub/pyLyA/issues/53

range limit of sigma 2 LSS is too low

### CODE REVIEW

**Comment by londumas** [comment]
Yes we can add a command line.
Do you want me to do it ?

**Comment by ngbusca** [comment]
Sure

**Comment by londumas** [comment]
done

**Comment by londumas** [comment]
@ngbusca is it ok for you ?

File: bin/do_deltas.py [hunk 1]
```diff
@@ -90,6 +90,16 @@ def cont_fit(data):
     parser.add_argument('--multiplicative-flux-calibration',type = str,default=None,required=False,
```
Comment (ngbusca) [inline bin/do_deltas.py]: I should have caught this before, but can you change this option to --flux-calib ? It's too long now...

**Comment by londumas** [comment]
done, you can merge

**Pull request merged**